### ISSUE TITLE
Create an interceptor that automatically debugs long running queries

### ISSUE THREAD

**Issue by gajus** [gajus/slonik#76]
It would be nice to have an interceptor that tracks query execution time and when it detects that query has been running for a pro-longed time, then it would dump additional information about the query, e.g. if it is being blocked by another query.

**Comment by gajus** [comment]
Issue out of scope for this repository.
